packages/mesh-extras/src/SimpleRope.ts

```diff
@@ -15,19 +15,16 @@ import type { IPoint } from '@pixi/math';
  * let rope = new PIXI.SimpleRope(PIXI.Texture.from("snake.png"), points);
  *  ```
  *
- * @class
- * @extends PIXI.Mesh
  * @memberof PIXI
- *
  */
 export class SimpleRope extends Mesh
 {
     public autoUpdate: boolean;
 
     /**
-     * @param {PIXI.Texture} texture - The texture to use on the rope.
-     * @param {PIXI.Point[]} points - An array of {@link PIXI.Point} objects to construct this rope.
-     * @param {number} [textureScale=0] - Optional. Positive values scale rope texture
+     * @param texture - The texture to use on the rope.
+     * @param points - An array of {@link PIXI.Point} objects to construct this rope.
+     * @param {number} textureScale - Optional. Positive values scale rope texture
```

Comment:
Why is `{number}` type necessary here? Shouldn't this infer from the default value?

Reply:
I opened a bug in webdoc - https://github.com/webdoc-labs/webdoc/issues/138. It's not inferring the data type based on the parameter's default value currently. (it expects `textureScale: number = 0`)